src/backend/impl/src/services/proposal_review_service.rs

```diff
@@ -652,6 +644,156 @@ impl<
+            })?;
+        let (id, current_proposal_review) = self
+            .proposal_review_repository
+            .get_proposal_review_by_proposal_id_and_user_id(proposal_id, user_id)
+            .ok_or_else(|| {
+                ApiError::not_found(&format!(
+                    "Proposal review for proposal {} for principal {} not found",
+                    raw_proposal_id,
+                    user_principal.to_text()
+                ))
+            })?;
+
+        Ok((id, current_proposal_review, proposal_id, user_id))
+    }
+
+    fn get_proposal_review_with_proposal_and_user_id(
+        &self,
+        raw_proposal_id: String,
+        user_principal: &Principal,
+    ) -> Result<(ProposalReviewId, ProposalReview, Proposal, UserId), ApiError> {
+        let (id, current_proposal_review, proposal_id, user_id) =
+            self.get_proposal_review_with_proposal_id_and_user_id(raw_proposal_id, user_principal)?;
+        let proposal = self
+            .proposal_repository
+            .get_proposal_by_id(&proposal_id)
+            .ok_or_else(|| {
+                // this should never happen, as the proposal review is associated with a proposal that must exist
+                ApiError::not_found(&format!(
+                    "Proposal with Id {} not found",
+                    proposal_id.to_string()
+                ))
+            })?;
+
+        Ok((id, current_proposal_review, proposal, user_id))
+    }
+}
+
+/// Returns the markdown representation for the proposal review.
+///
+/// Template:
+/// ```markdown
+/// # Proposal [nervous system proposal id]
+///
+/// Hashes match: [true or false]
+/// All reviewed commits match their descriptions: [true or false]
+///
+/// [proposal review images if any]
+///
+/// Summary:
+/// [proposal review summary if any]
+///
+/// Commits review:
+/// - **[commit sha truncated to 9 characters]**:
+///   Matches description: [true or false]
+///   Comment: [commit comment]
+///   Highlights: [comma-separated list of commit highlights]
+/// - **[commit sha truncated to 9 characters]**:
+///   ...
+/// ...
+/// ```
+fn proposal_review_summary_markdown(
+    proposal: &Proposal,
+    proposal_review: &ProposalReview,
+    proposal_review_commits: &[(ProposalReviewCommitId, ProposalReviewCommit)],
+    images_paths: &[String],
+) -> String {
+    let mut md_content = String::new();
+    let reviewed_commits: Vec<&(ProposalReviewCommitId, ProposalReviewCommit)> =
+        proposal_review_commits
+            .iter()
+            .filter(|(_, commit)| commit.is_reviewed())
+            .collect();
+
+    // header
+    {
+        md_content.push_str(&format!(
+            "# Proposal {}\n\n",
+            proposal.nervous_system.proposal_id()
+        ));
+    }
+    // info
+    {
+        md_content.push_str(&format!("Vote: {}\n", proposal_review.vote));
+        md_content.push_str(&format!(
+            "Hashes match: {}\n",
+            proposal_review.build_reproduced.unwrap_or(false)
```

Comment:
Similarly here, maybe it's better to be explicit and put "Unanswered" or something similar.

Reply:
Fixed